stdlib/sched.pyi

```diff
@@ -33,12 +36,8 @@ class scheduler:
     delayfunc: Callable[[float], object]
 
     def __init__(self, timefunc: Callable[[], float] = ..., delayfunc: Callable[[float], object] = ...) -> None: ...
-    def enterabs(
-        self, time: float, priority: Any, action: _ActionCallback, argument: tuple[Any, ...] = (), kwargs: dict[str, Any] = ...
-    ) -> Event: ...
-    def enter(
-        self, delay: float, priority: Any, action: _ActionCallback, argument: tuple[Any, ...] = (), kwargs: dict[str, Any] = ...
-    ) -> Event: ...
+    def enterabs(self, time: float, priority: object, action: Callable[PS, R], *args: PS.args, **kwargs: PS.kwargs) -> Event: ...
+    def enter(self, delay: float, priority: object, action: Callable[PS, R], *args: PS.args, **kwargs: PS.kwargs) -> Event: ...
```

Comment:
These changes **_are wrong_**. I'd _like_ to be able to say something like the following, but (afaik) Python's type system doesn't support it[^1] :(

```suggestion
    def enterabs(self, time: float, priority: Any, action: Callable[PS, R], argument: PS.args, kwargs: PS.kwargs) -> Event: ...
    def enter(self, delay: float, priority: Any, action: Callable[PS, R], argument: PS.args, kwargs: PS.kwargs) -> Event: ...
```

[^1]: or at least, type checkers do not support it